### ISSUE TITLE
JFrog CLI task doesn't work as expected

### ISSUE THREAD

**Issue by vyckou** [jfrog/artifactory-azure-devops-extension#189]
**Describe the bug**
Not all `jfrog rt` commands do support `--url`, `--user` and `--password` arguments

**To Reproduce**
Add the JFrog CLI following task in azure devops pipeline and try to execute 

```
- task: JfrogCli@1
  name: artifactory_configuration
  inputs:
    artifactoryService: 'Artifactory'
    useCustomVersion: true
    cliVersion: '1.37.1'
    command: 'jfrog rt dotnet-config --repo-resolve joined-nuget-buildfeed --server-id-resolve Default-Server'

```
As a result you will get
```
##[section]Starting: artifactory_configuration_2
==============================================================================
Task         : JFrog CLI
Description  : Run your own JFrog CLI command.
Version      : 1.10.0
Author       : JFrog
Help         : [Learn more about JFrog CLI](https://www.jfrog.com/confluence/display/CLI/JFrog+CLI)
==============================================================================
Found tool in cache: jfrog 1.37.1 x64
Running jfrog-cli from /__t/jfrog/1.37.1/x64/jfrog.
JFrog CLI version: 1.37.1
Incorrect Usage: flag provided but not defined: -url
[Error] flag provided but not defined: -url


Name:
  jfrog rt dotnet-config - Generate dotnet configuration.

Usage:
  jfrog rt dotnet-config [command options]

Options:
  --global               [Default: false] Set to true if you'd like the configuration to be global (for all projects). Specific projects can override the global configuration.
  --server-id-resolve    [Optional] Artifactory server ID for resolution. The server should configured using the 'jfrog rt c' command.
  --repo-resolve         [Optional] Repository for dependencies resolution.
  
Environment Variables:
  JFROG_CLI_LOG_LEVEL

.....

  CI
    [Default: false]
    If true, disables interactive prompts and progress bar.

##[error]Error: Command failed: /__t/jfrog/1.37.1/x64/jfrog rt dotnet-config --repo-resolve joined-nuget-buildfeed --server-id-resolve Default-Server --url="https://artifactory.someorg.net/artifactory" --user=*** --password=***
##[section]Finishing: artifactory_configuration_2
```

**Expected behavior**
cli should restore nuget packages

**Versions**
- Artifactory Azure DevOps extension version: 1.10 
- Agent operating system: MacOS/Linux
- JFrog Artifactory version: 6.21

**Additional context**
https://github.com/jfrog/jfrog-cli/blob/master/utils/cliutils/commandsflags.go#L1058
you can find  arguments, which are accepted by any of the command
you cannot blindly attach url, user and password to any `jfrog rt` given command at
https://github.com/jfrog/artifactory-azure-devops-extension/blob/master/tasks/JfrogCli/jfrogCliRun.js#L39

any "switch" within the task attach or not attach url/user/password - would be very nice

**Comment by lindhe** [comment]
Somewhat related to #186

**Comment by RobiNino** [comment]
Hi @vyckou , @lindhe ,
Im closing this issue as it should be resolved in release 1.11.0.
Do let us know if you have any new feedback about the matter.
Thanks!
